content/workshops/ansible_tower/exercise1.4.adoc

```diff
@@ -158,15 +183,21 @@ An Inventory can also be imported into Tower using the `tower-manage` command
 and this is how we are going to add an inventory for this workshop.
 
 
-*Step 1:* Click on INVENTORIES +
+=== Step 1:
+
+Click on INVENTORIES +
```

Comment:
You don't need the trailing `+` here. The extra line below this forces the new line.